GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitCommandsTests.cs

```diff
@@ -1019,6 +1019,18 @@ public void EditFileNeedingUtf8Encoding()
             GVFSHelpers.ModifiedPathsShouldContain(this.Enlistment, this.FileSystem, relativeGitPath);
         }
 
+        [TestCase]
+        [Category(Categories.NewTest)]

```

Comment:
Remove `[Category(Categories.NewTest)]`?  (I am guessing it's debug code that was missed during cleanup)

Reply:
Disregard, this was from an earlier push